src/alerts/Alerts.test.ts

```diff
@@ -0,0 +1,81 @@
+import { fireEvent, screen, within } from '@testing-library/react';
+import { h } from 'react-hyperscript-helpers';
+import * as Utils from 'src/libs/utils';
+import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
+
+import { Alert as AlertT } from './Alert';
```

Comment:
as AlertType to be clearer?

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4400/commits/0958f5e50988c73c9bdcf0f48a0024dadf19c118.